device_tests/test_minimi_400.py

```diff
@@ -0,0 +1,212 @@
+import argparse
```

Comment:
Can you document what this script is for

Reply:
added documentation in the script itself